content/videos/challenges/54.2-star-patterns-update-law-of-sines/index.json

```diff
@@ -1,28 +1,37 @@
 {
     "title": "Islamic Star Patterns",
-    "description": "In this bonus super-sized coding challenge, I work through visualizing Islamic Star Patterns in p5.js based on this University of Waterloo paper",
-    "videoNumber": "54.1",
-    "videoId": "sJ6pMLp_IaI",
-    "date": "2017-02-01",
+    "description": "In this bonus super-sized coding challenge, I work through visualizing the Islamic Star Patterns in p5.js.",
+    "videoNumber": "54.2",
+    "videoId": "lobJ9gzbLo8",
+    "date": "2017-02-14",
     "languages": ["p5.js", "javascript"],
     "topics": [
         "JavaScript",
```

Comment:
Maybe from here on out, anything listed in `languages` should be left out of `topics`